Modules/Phasing.lua

```diff
@@ -665,12 +665,16 @@ function Phasing.IsSpawnVisible(phase)
         return (not complete[27769])
     end
 
+    if phase == phases.RHEA_NEW_KARGATH then
+        return complete[27769] and (questLog[27889] and questLog[27889].isComplete == 1) and ((not complete[27897]) and ((not questLog[27897]) or (questLog[27897] and questLog[27897].isComplete == 0))) or false
+    end
+
     if phase == phases.RHEA_DRAGONS_MOUTH then
-        return complete[27769] and ((not complete[27832]) and ((not questLog[27832]) or (questLog[27832] and questLog[27832].isComplete == 0))) or false
+        return complete[27769] and (questLog[27826] and questLog[27826].isComplete == 1) and ((not complete[27832]) and ((not questLog[27832]) or (questLog[27832] and questLog[27832].isComplete == 0))) or false
```

Comment:
Same as with the other `27826` has no objective, so the `isComplete` check is obsolete.

```suggestion
        return complete[27769] and questLog[27826] and ((not complete[27832]) and ((not questLog[27832]) or (questLog[27832] and questLog[27832].isComplete == 0))) or false
```

Reply:
@BreakBB thanks for reminder - not resolved yet. Going to go through again with fresh character to test

Reply:
@BreakBB I have made further changes below (more detail on Discord) and think this is sorted now.